common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -86,6 +86,45 @@
 source, template_path = Loader(engine).load_template_source(path)
 %>${source | n, decode.utf8}</%def>
 
+<%def name="studiofrontend(page, lang='en')">
+    <%doc>
+        Loads a studio-frontend page, with the necessary context. Context is expected
+        as a dictionary in the body of this tag.
+
+        Dev note: we could also add the locale-injection script in this block
+            -use a better default than the hardcoded 'en'. There should be a setting or something?
+            -lookup (webpack exported) locale-injection script using lang as key
+            -include it as the first script in this block
```

Comment:
Normally I'm against merging TODO-style comments in code, but since this is a new template tag specifically for studio-frontend and these comments are relevant to tickets that are coming up soon, I'm 👍 on this dev note staying put.